app/assets/stylesheets/components/_share-links.scss

```diff
@@ -1,20 +1,21 @@
-$share-button-width: 25px;
-$share-button-height: 25px;
+$share-button-width: 32px;
+$share-button-height: 32px;
 
 .app-c-share-links__link {
   margin-right: $gutter;
-  @include core-16;
+  @include bold-16($line-height: $share-button-height, $line-height-640: $share-button-height);
+  text-decoration: none;
 
-  &:visited .app-c-share-links__link__icon {
-    background-color: $link-visited-colour;
+  &:visited .app-c-share-links__link__icon .background {
+    fill: $link-visited-colour;
   }
 
-  &:active .app-c-share-links__link__icon {
-    background-color: $link-active-colour;
+  &:active .app-c-share-links__link__icon .background {
+    fill: $link-active-colour;
   }
 
-  &:hover .app-c-share-links__link__icon {
-    background-color: $link-hover-colour;
+  &:hover .app-c-share-links__link__icon  .background {
+    fill: $link-hover-colour;
```

Comment:
This only passes colour contrast for large text (this is already an issue we have with all links on GOV.UK I believe though):

<img width="716" alt="screen shot 2017-12-04 at 17 24 50" src="https://user-images.githubusercontent.com/29889908/33566572-4cf3a29c-d918-11e7-9bb0-bee087086f5d.png">

Reply:
Since the icon is backed up by a good label, these icons are presentation and therefore don't need to pass contrast tests.